chicagoland.pandemicresponsecommons.org/portal/gitops.json

```diff
@@ -125,68 +125,107 @@
     "analysis": true
   },
   "analysisTools": [ ],
-  "dataExplorerConfig": {
-    "charts": {
-      "project_id": {
-        "chartType": "count",
-        "title": "Datasets"
+  "explorerConfig": [
+    {
+      "tabTitle": "Datasets",
+      "filters": {
+        "tabs": [
+          {
+            "title": "Catalogue",
+            "fields": [
+              "program_name",
+              "code"
+            ]
+          }
+        ]
       },
-      "symptoms": {
-        "chartType": "bar",
-        "title": "Symptoms"
+      "table": {
+        "enabled": true,
+        "fields": [
+          "name",
+          "data_description",
+          "code"
+        ]
       },
-      "source": {
-        "chartType": "pie",
-        "title": "Sources"
+      "buttons": [],
+      "guppyConfig": {
+        "dataType": "dataset",
```

Comment:
is this index ready? i don't see it in the manifest

Reply:
https://github.com/uc-cdis/cdis-manifest/blob/master/chicagoland.pandemicresponsecommons.org/manifest.json#L182

Reply:
uh... missed that one. thanks